### ISSUE TITLE
segmention fault（core dump）

### ISSUE THREAD

**Issue by wstecho** [sunnyxiaohu/R-C3D.pytorch#13]
Hi, thanks for your work. When I run trainval_net.py, I met segment fault（core dump） error as follows.
I use pytorch0.41 and RTX2080ti, 2080ti cannot install pytorch0.4
Could you please help me to solve this issue? Thank you.

Called with args:
Namespace(batch_size=1, checkepoch=1, checkpoint=13711, checkpoint_interval=10000, checksession=1, dataset='thumos14', disp_interval=100, gpus=None, lr=0.0001, lr_decay_gamma=0.1, lr_decay_step=3, max_epochs=5, net='c3d', num_workers=5, optimizer='sgd', resume=False, roidb_dir='./preprocess', save_dir='./models/20181115', session=1, start_epoch=1, use_tfboard=False)
Using config:
{'ANCHOR_SCALES': [2, 4, 5, 6, 8, 9, 10, 12, 14, 16],
 'CROP_RESIZE_WITH_MAX_POOL': True,
 'CUDA': True,
 'DATA_DIR': '/home/wst/hdd/R-C3D.pytorch/data',
 'DEDUP_TWINS': 0.125,
 'EPS': 1e-14,
 'EXP_DIR': 'default',
 'FEAT_STRIDE': [8],
 'FPS': 25,
 'GPU_ID': 0,
 'INPUT': 'video',
 'MATLAB': 'matlab',
 'MAX_NUM_GT_TWINS': 20,
 'MOBILENET': {'DEPTH_MULTIPLIER': 1.0,
               'FIXED_LAYERS': 5,
               'REGU_DEPTH': False,
               'WEIGHT_DECAY': 4e-05},
 'NUM_CLASSES': 21,
 'PIXEL_MEANS': array([[[ 90,  98, 102]]]),
 'PIXEL_MEANS_FLOW': array([128]),
 'POOLING_HEIGHT': 2,
 'POOLING_LENGTH': 4,
 'POOLING_MODE': 'pool',
 'POOLING_WIDTH': 2,
 'RESNET': {'FIXED_BLOCKS': 0, 'MAX_POOL': False},
 'RNG_SEED': 3,
 'ROOT_DIR': '/home/wst/hdd/R-C3D.pytorch',
 'RPN_HAS_MASK': False,
 'TEST': {'HAS_RPN': True,
          'MODE': 'nms',
          'NMS': 0.4,
          'PROPOSAL_METHOD': 'gt',
          'RPN_MIN_SIZE': 0,
          'RPN_NMS_THRESH': 0.9,
          'RPN_POST_NMS_TOP_N': 300,
          'RPN_PRE_NMS_TOP_N': 6000,
          'RPN_TOP_N': 5000,
          'TWIN_REG': True},
 'TRAIN': {'BATCH_SIZE': 128,
           'BG_THRESH_HI': 0.5,
           'BG_THRESH_LO': 0.1,
           'BIAS_DECAY': False,
           'BN_TRAIN': False,
           'CINPUT': False,
           'CROP_SIZE': 112,
           'DISPLAY': 10,
           'DOUBLE_BIAS': True,
           'FG_FRACTION': 0.5,
           'FG_THRESH': 0.5,
           'FRAME_SIZE': [128, 171],
           'GAMMA': 0.1,
           'HAS_RPN': True,
           'IMS_PER_BATCH': 1,
           'LEARNING_RATE': 0.001,
           'LENGTH': [768],
           'MOMENTUM': 0.9,
           'OHEM_LOSS_ONLY_CLASSIFICATION': False,
           'OHEM_LOSS_ONLY_LOCALIZATION': False,
           'OHEM_NMS_THRESH': 0.7,
           'OHEM_USE_NMS': True,
           'PROPOSAL_METHOD': 'gt',
           'RANDOM': False,
           'RPN_BATCHSIZE': 64,
           'RPN_CLOBBER_POSITIVES': False,
           'RPN_FG_FRACTION': 0.5,
           'RPN_HAS_MASK': False,
           'RPN_MIN_SIZE': 0,
           'RPN_NEGATIVE_OVERLAP': 0.3,
           'RPN_NMS_THRESH': 0.8,
           'RPN_POSITIVE_OVERLAP': 0.7,
           'RPN_POSITIVE_WEIGHT': -1.0,
           'RPN_POST_NMS_TOP_N': 2000,
           'RPN_PRE_NMS_TOP_N': 12000,
           'RPN_TWIN_INSIDE_WEIGHTS': [1.0, 1.0],
           'SNAPSHOT_ITERS': 10000,
           'SNAPSHOT_KEPT': 3,
           'SNAPSHOT_PREFIX': 'res101_faster_rcnn',
           'STEPSIZE': [30000],
           'SUMMARY_INTERVAL': 180,
           'TRUNCATED': False,
           'TWIN_INSIDE_WEIGHTS': [1.0, 1.0],
           'TWIN_NORMALIZE_MEANS': [0.0, 0.0],
           'TWIN_NORMALIZE_STDS': [0.1, 0.2],
           'TWIN_NORMALIZE_TARGETS': True,
           'TWIN_NORMALIZE_TARGETS_PRECOMPUTED': True,
           'TWIN_REG': True,
           'TWIN_THRESH': 0.5,
           'USE_ALL_GT': True,
           'USE_FLIPPED': True,
           'VIDEO_BATCH': 1,
           'WEIGHT_DECAY': 0.0005},
 'USE_GPU_NMS': True}
13712 roidb entries
c3d_tdcnn(
  (RCNN_rpn): _RPN(
    (RPN_Conv1): Conv3d(512, 512, kernel_size=(3, 3, 3), stride=(1, 2, 2), padding=(1, 1, 1))
    (RPN_Conv2): Conv3d(512, 512, kernel_size=(3, 3, 3), stride=(1, 2, 2), padding=(1, 1, 1))
    (RPN_output_pool): MaxPool3d(kernel_size=(1, 2, 2), stride=(1, 2, 2), padding=0, dilation=1, ceil_mode=False)
    (RPN_cls_score): Conv3d(512, 20, kernel_size=(1, 1, 1), stride=(1, 1, 1))
    (RPN_twin_pred): Conv3d(512, 20, kernel_size=(1, 1, 1), stride=(1, 1, 1))
    (RPN_proposal): _ProposalLayer()
    (RPN_anchor_target): _AnchorTargetLayer()
  )
  (RCNN_proposal_target): _ProposalTargetLayer()
  (RCNN_roi_temporal_pool): _RoITemporalPooling()
  (RCNN_base): Sequential(
    (0): Conv3d(3, 64, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1))
    (1): ReLU(inplace)
    (2): MaxPool3d(kernel_size=(1, 2, 2), stride=(1, 2, 2), padding=0, dilation=1, ceil_mode=False)
    (3): Conv3d(64, 128, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1))
    (4): ReLU(inplace)
    (5): MaxPool3d(kernel_size=(2, 2, 2), stride=(2, 2, 2), padding=0, dilation=1, ceil_mode=False)
    (6): Conv3d(128, 256, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1))
    (7): ReLU(inplace)
    (8): Conv3d(256, 256, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1))
    (9): ReLU(inplace)
    (10): MaxPool3d(kernel_size=(2, 2, 2), stride=(2, 2, 2), padding=0, dilation=1, ceil_mode=False)
    (11): Conv3d(256, 512, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1))
    (12): ReLU(inplace)
    (13): Conv3d(512, 512, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1))
    (14): ReLU(inplace)
    (15): MaxPool3d(kernel_size=(2, 2, 2), stride=(2, 2, 2), padding=0, dilation=1, ceil_mode=False)
    (16): Conv3d(512, 512, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1))
    (17): ReLU(inplace)
    (18): Conv3d(512, 512, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1))
    (19): ReLU(inplace)
  )
  (RCNN_top): Sequential(
    (0): Linear(in_features=8192, out_features=4096, bias=True)
    (1): ReLU(inplace)
    (2): Dropout(p=0.5)
  )
  (RCNN_cls_score): Linear(in_features=4096, out_features=21, bias=True)
  (RCNN_twin_pred): Linear(in_features=4096, out_features=42, bias=True)
)
RCNN_rpn.RPN_Conv1.weight
RCNN_rpn.RPN_Conv1.bias
RCNN_rpn.RPN_Conv2.weight
RCNN_rpn.RPN_Conv2.bias
RCNN_rpn.RPN_cls_score.weight
RCNN_rpn.RPN_cls_score.bias
RCNN_rpn.RPN_twin_pred.weight
RCNN_rpn.RPN_twin_pred.bias
RCNN_base.6.weight
RCNN_base.6.bias
RCNN_base.8.weight
RCNN_base.8.bias
RCNN_base.11.weight
RCNN_base.11.bias
RCNN_base.13.weight
RCNN_base.13.bias
RCNN_base.16.weight
RCNN_base.16.bias
RCNN_base.18.weight
RCNN_base.18.bias
RCNN_top.0.weight
RCNN_top.0.bias
RCNN_cls_score.weight
RCNN_cls_score.bias
RCNN_twin_pred.weight
RCNN_twin_pred.bias
segment fault（core dump）

**Comment by Fordacre** [comment]
@wstecho Hello, have you solved it? I faced the same problem.

**Comment by Fordacre** [comment]
Sovled it by setting cfg.USE_GPU_NMS = False.
